### ISSUE TITLE
Maintain helm chart

### ISSUE THREAD

**Issue by raoofm** [coreos/etcd-operator#1355]
helm chart is outdated, is it being maintained or has a plan to maintain by etcd-operator team.

for example

- etcd is outdated 3.1.8
- etcd-operator is outdated 0.3.3
- it still uses tpr not crd

**Comment by hongchaodeng** [comment]
No.
We don't have the bandwidth to maintain it at the moment.

**Comment by xiang90** [comment]
@raoofm @hongchaodeng 

i labeled this as future. it is a good thing to do, but not high priority for now.

**Comment by raoofm** [comment]
@xiang90 thnx, was just checking as it was confusing to refer the chart and the examples here.

**Comment by raoofm** [comment]
@xiang90 @hongchaodeng I assumed it is being maintained as it was referred here https://github.com/coreos/etcd-operator/blob/master/doc/user/op_guide.md#installation-via-helm without any warning of usage :)

**Comment by Nitrogenycs** [comment]
In case you do not maintain the chart anymore, please put a message in the readme stating so to save people wasting time.

I've spent quite a few hours now getting it working. It breaks completely with kubectl v1.8 due to the use of TPRs and it also does not spawn a cluster, i.e. cluster.enabled=true does not work. This breaks my helm setup.

**Comment by xiang90** [comment]
we never try to maintain it. someone sent us a pr to add it there.

we can:

1. make it clear that it is an externally maintained project
2. remove it from etcd-operator reference.

@Nitrogenycs 

you should file a PR to where the chart is. The maintainer is probably more active there.

**Comment by raoofm** [comment]
@Nitrogenycs you are right that chart is kind of broken or it isn't the right way of doing it. I had spent a day as well thinking that is an approved chart as I found it in etcd-operator guide and figuring out what am I missing as it was in the stable directory. Probably it should be moved to incubator again.

> we never try to maintain it

@xiang90 it would be a great help to have it there. I even gave it a shot but the spec changed quite frequently. I think option 1 is good so that at least if any user wants to maintain it they see the link in the project.
